lib/settings.js

```diff
@@ -157,16 +157,6 @@ async function setReduceMotion (sim, reduceMotion = true) {
   }
 }
 
-async function setDarkMode (sim, isEnabled = true) {
```

Comment:
Will we remove this logic? (which can work on before Xcode 11.3)

Reply:
I think it would be better to get rid of it since it only works with limited set of environments and also is not used anywhere outside of this module

Reply:
_not used anywhere outside of this module_
Ah... I did not know this.

Seems safe to remove it 👍